sklearn/isotonic.py

```diff
@@ -168,6 +168,14 @@ class IsotonicRegression(RegressorMixin, TransformerMixin, BaseEstimator):
     X_max_ : float
         Maximum value of input array `X_` for right bound.
 
+    X_thresholds_ : array-like of shape (n_thresholds,)
+        De-duplicated ascending `X` values suitable to interpolate
+        the y = f(X) monotonic function.
+
+    y_thresholds_ : array-like of shape (n_thresholds,)
+        De-duplicated ascending `y` values suitable to interpolate
+        the y = f(X) monotonic function.
```

Comment:
Looking at the tests, x_thresholds is always ascending but `y_thresholds` depends on the increasing param.